libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat

```diff
@@ -147,3 +147,12 @@ ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
 
 define BOARD_PWM_COUNT_DEFAULT 6
 define STM32_PWM_USE_ADVANCED TRUE
+
+# disable SMBUS and fuel battery monitors to save flash
```

Comment:
Unrelated changes; should be PR'd separately.

That does run the risk of someone swiping "your" space, of course....

Reply:
this is an unwanted leftover but it is related, without it bidir stuff does not fit

Reply:
but it does not belong to this PR, 1MB board support is "another story"